### ISSUE TITLE
Disable Animation in Settings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tripodtr/npr-android-app#46]
```
It would be nice if users with slower phones could disable animation from 
Settings. Alternatively, this could just be keyed off of the Disable Animation 
setting global to the Android OS.
```

Original issue reported on code.google.com by `soopah...@gmail.com` on 8 Oct 2010 at 11:07

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Agreed.

patch attached
```

Original comment by `tjack1...@gmail.com` on 1 Dec 2010 at 5:08

Attachments:
- [disable-transition-animations](https://storage.googleapis.com/google-code-attachments/npr-android-app/issue-46/comment-2/disable-transition-animations)

**Comment by GoogleCodeExporter** [comment]
```
Have you sent this around for a code review? I haven't seen it. Don't want any 
of your work to get left out if we can use it.
```

Original comment by `jpenn...@gmail.com` on 6 Jan 2011 at 7:54
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
tjack - if you post this to codereview.appspot.com, per the wiki, it would be 
easier for all of us to comment on it.

Off hand, the 'if' at line 33 need backets to match standard.

Also, I will point out that this applies to version_1.x branch, not trunk, 
because trunk no longer has the BackAndForthActivityGroup and I think that 
animations have been mostly removed in that version anyway. I don't know if 
this is priority high-enough to merit pushing to a 1.x release. Depends on 
those testing the release.
```

Original comment by `jeremy.w...@gmail.com` on 6 Jan 2011 at 8:33

**Comment by GoogleCodeExporter** [comment]
```
For 2.0 this patch is no longer applicable. The animations are mostly removed, 
although the navigation menu still animates (when default system animation is 
enabled). 

Note that while the app has code to not animate transition on many actions this 
only works on 1.6 and later. And probably it's the 1.5 and 1.6 devices that 
would want this most.

I feel this has been addressed in 2.0 but would appreciate more testing from 
users with slower phones to validate.
```

Original comment by `jeremy.w...@gmail.com` on 23 Mar 2011 at 4:25

**Comment by GoogleCodeExporter** [comment]
```
Closing this for above reasons from Wadsack.
```

Original comment by `jpenn...@gmail.com` on 19 May 2011 at 7:33
- Changed state: **WontFix**
